doc/nrf/ug_matter_configuring_env.rst

```diff
@@ -173,16 +217,25 @@ To configure and use Thread Border Router and Matter controller on the same devi
 1. Program the Matter accessory device with one of available :ref:`matter_samples`.
 #. Configure the Thread Border Router on a PC or on a Raspberry Pi, depending on what hardware you are using.
    For detailed steps, see the Running OTBR using Docker section on the :ref:`ug_thread_tools_tbr` page in the |NCS| documentation.
-#. Configure the Python Matter controller:
+#. Configure the controller, depending on the controller you want to use:
 
-   a. Build the controller using one of the options described in :ref:`ug_matter_configuring_controller_pc`.
-   #. Complete the following steps from the :doc:`matter:python_chip_controller_building` user guide in the Matter documentation:
+   * CHIP Tool controller
 
-      * Building and installing - which prepares your CHIPTool for commissioning.
-      * Running the tool - which gives you access to Bluetooth interface.
-      * Using Python CHIP Controller for Matter accessory testing - which prepares your device programmed with the Matter sample for commissioning and testing.
+      a. Complete the following the steps from the sections of the :doc:`matter:chip_tool_guide` in the Matter documentation:
 
-        .. note::
-            Skip Step 1, as you have the development kit programmed with a Matter sample.
+         * Build and run the CHIP Tool by completing the steps listed in "Building and running CHIP Tool".
+         * Prepare the environment for testing by completing the steps listed in "Using CHIP Tool for Matter device testing".
+
+   * Python Matter controller:
+
+      a. Build the controller using one of the options described in :ref:`ug_matter_configuring_controller_pc`.
+      #. Complete the following steps from the :doc:`matter:python_chip_controller_building` user guide in the Matter documentation:
+
+         * Building and installing - which prepares your CHIPTool for commissioning.
+         * Running the tool - which gives you access to Bluetooth interface.
```

Comment:
```suggestion
         * Running the tool - which opens the command line interface
```
?
BLE is just one of many features of the tool, so we shouldn't suggest it's the primary feature or so.